### PR TITLE
Stream_support:  Add example to read an xml file using boost::property_tree

### PR DESCRIPTION
**Pull request by afabri**
## Summary of Changes

Add a small subsection with a 10liner example for a typical non-standard adhoc xml file,
as it needs the documentation of boost + stackoverflow to see how to parse attributes.  
## Release Management

* Affected package(s): Stream_support

### CODE REVIEW

**Comment by afabri** [comment]
@maxGimeno ready for testing.

**Comment by afabri** [comment]
Successfully tested in Ic. 
Example is in the [User Manual](https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-5.0-Ic-43/output1/Stream_support/index.html#title13)

File: Stream_support/doc/Stream_support/IOstream.txt [hunk 1]
```diff
@@ -258,6 +258,24 @@ in those WKT types, namely:
 - `CGAL::Point_3`
 - `CGAL::Polygon_with_holes_2`
 - random access range of the above types.
+
+\section IOstreamXML Reading Adhoc XML
+
+In case you have to read data in a simple xml file with a simple structure, without existing I/O libraries
+the <a href="https://www.boost.org/doc/libs/property_tree">`boost::property_tree`</a> comes in handy.
```
Comment (lrineau) [inline Stream_support/doc/Stream_support/IOstream.txt]: This links to a 404 error.
https://www.boost.org/doc/libs/property_tree

Comment (lrineau) [reply Stream_support/doc/Stream_support/IOstream.txt]: I have fixed the URL in https://github.com/CGAL/cgal/pull/3931/commits/78c5ec760f982d936f2a22fc0926a921c76b3546.

**Comment by lrineau** [comment]
@maxGimeno In which testsuite was that tested?

**Comment by maxGimeno** [comment]
Sorry, forgot to paste my concluding message :/ 
It was tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.0-Ic-51.shtml

**Pull request merged**